ev3dev/motor.py

```diff
@@ -112,36 +219,10 @@ def __init__(self, address=None, name_pattern=SYSTEM_DEVICE_NAME_CONVENTION, nam
         self._time_sp = None
         self._poll = None
 
-    __slots__ = [
-    '_address',
-    '_command',
-    '_commands',
-    '_count_per_rot',
-    '_count_per_m',
-    '_driver_name',
-    '_duty_cycle',
-    '_duty_cycle_sp',
-    '_full_travel_count',
-    '_polarity',
-    '_position',
-    '_position_p',
-    '_position_i',
-    '_position_d',
-    '_position_sp',
-    '_max_speed',
-    '_speed',
-    '_speed_sp',
-    '_ramp_up_sp',
-    '_ramp_down_sp',
-    '_speed_p',
-    '_speed_i',
-    '_speed_d',
-    '_state',
-    '_stop_action',
-    '_stop_actions',
-    '_time_sp',
-    '_poll',
-    ]
+        # We use poll() to monitor the state of the motor so we can react when
+        # the state changes but if you have never read the file, poll() won't
+        # block. Read 'state' once so that poll() will block.
+        self.state
```

Comment:
This is only necessary if you are not calling poll in a loop, otherwise the loop does exactly the same thing.

Reply:
Seems worth keeping though, it is only one read and if the user does their own thing with POLLPRI they won't get tripped up on this.

Reply:
The user will still have to read once themselves so that they know the initial state

Reply:
Also, I just realized that polling the state attribute will only work on EV3 (and EVB since they share hardware drivers), but it won't work on any other platform (BrickPi, BrickPi3, PiStorms).

Reply:
will back this part out

Any ideas on how to get poll() working on BrickPi, etc?